scripts/create_taginfo_items.py

```diff
@@ -10,4 +10,4 @@
   for value in values:
     items.append({'key':key,'value':value,'object_types':argv[5].split(','),'description':argv[6]})
 
-print(json.dumps(items))
+print((json.dumps(items)))
```

Comment:
these kinds of changes make no sense to me

Reply:
yeah i  think this happens because `2to3` thinks `print` (without `print_function` import) is printing a tuple and encapsulates it in another set of `()`. I think using `2to3 -p` will fix the issue. Let me try that.